policies/Sponsorship_Policy.md

```diff
@@ -0,0 +1,153 @@
+# Sponsorship Policy
+
+## Purpose
+
+The purpose of the Sponsorship Policy (The Policy) is to outline the principles
+ and behaviours adopted by OpenSSL when receiving or providing sponsorship.  
+
+The Policy will ensure that only appropriate sponsorship is received or 
+provided by OpenSSL, i.e. that sponsorship aligns with OpenSSL's Mission and values, 
+community expectations and meets legal requirements.
+
+
+## Scope
+
+The policy applies to all Organisations, individuals and entities that provide 
+sponsorship to OpenSSL and to any organisation, individual or entity that 
+OpenSSL sponsors.
+
+The Policy establishes a framework by which new and renewing sponsorships are 
+assessed, formalised, managed, and measured for both the sponsorship of OpenSSL
+ and any sponsorship that OpenSSL may provide.  
```

Comment:
leading space at start of line isn't required (and confuses markdown at times)

Reply:
Fixed